nat/traversal/pinger.go

```diff
@@ -115,7 +123,7 @@ func (p *Pinger) Stop() {
 
 // PingProvider pings provider determined by destination provided in sessionConfig
 func (p *Pinger) PingProvider(ip string, localPorts, remotePorts []int, proxyPort int) (localPort, remotePort int, err error) {
-	conns, err := p.PingPeer(ip, localPorts, remotePorts, maxTTL, 1)
+	conns, err := p.PingProviderPeer(ip, localPorts, remotePorts, maxTTL, 1)
```

Comment:
Is it ok to use maxTTL from the beginning? maxTTL should be set only if we receive message from peer.

Now, provider's router might treat incoming message from consumer as ddos and disable this consumer for some time. Even if provider sends message to this consumer, router keeps this consumer disabled for some time. We need incremental ttl here too.

Reply:
This is a consumer part that sets maxTTL from the very beginning. 
The provider starts listening and sending pings before sending config to the consumer. 
So consumer send first ping when provider router already expecting it.